modules/gapi/misc/python/test/test_gapi_core.py

```diff
@@ -30,7 +30,7 @@ def test_add(self):
         g_in1 = cv.GMat()
         g_in2 = cv.GMat()
         g_out = cv.gapi.add(g_in1, g_in2)
-        comp = cv.GComputation(g_in1, g_in2, g_out)
+        comp = cv.GComputation(cv.GIn(g_in1, g_in2), cv.GOut(g_out))
```

Comment:
why not `cv.gin` and `cv.gout`?

Reply:
Because `cv.gin & cv.gout` using for `cv::Mat`, `cv::Scalar` and etc whereas `cv.GIn & cv.GOut` using for `GMat`, `GScalar` ...